presto-main/src/main/java/com/facebook/presto/operator/OperatorExecutionContext.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator;
+
+import com.facebook.presto.execution.Lifespan;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import static java.util.Objects.requireNonNull;
+
+public class OperatorExecutionContext
```

Comment:
This object is also used in `DeleteOperator`, so maybe name it `OperatorExecutionInfo`?

Reply:
`OperatorExecutionInfo` might suggests it's general to any operators. `TableOperatorExecutionInfo`?  I am not sure...

Reply:
It could be general to any operators but since we added `CommitGranularity`, it might only be feasible for `TableFinishOperator` to consume it (or we make `CommitGranularity` to be `Optional`). I don't know if this sounds like a good starting point though :|

Reply:
`TableFinishContext`? `TableCommitContext`?

Reply:
For the two options you raised, I prefer `TableCommitContext` :D

Reply:
thoughts ? @arhimondr , @rschlussel ?

Reply:
Per offline discussion, let's go with `TableCommitContext`.